ssh/foxpass-keygen.py

```diff
@@ -29,6 +29,7 @@
 import getpass
 import json
 import os
+import stat
```

Comment:
does this get used?

Reply:
Nope, was going to but could resolve it with an octal notation instead.  Good find